### PR TITLE
Wizard recipe: libbensolve-v2.1.0

### PR DESCRIPTION
**Pull request by kofgokhan**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: libbensolve
* Version: v2.1.0

@staticfloat please review and merge.

### CODE REVIEW

File: L/libbensolve/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ArchiveSource("https://users.fmi.uni-jena.de/~sa67leb/bensolve/files/bensolve-$(version).tgz", "f13c6708974553e346c5426c0471f395fc4de417b55ac9fb07978fa9d268f0a0")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/bensolve*/
+gcc -shared -fPIC -std=c99 -O3 -o libbensolve.$dlext *.c -lglpk -lm
+install_license doc/manual.pdf
+cp libbensolve.$dlext ${libdir}
+cp *.h ${includedir}
```
Comment (giordano) [inline L/libbensolve/build_tarballs.jl:19]:
```suggestion
cc -shared -fPIC -std=c99 -O3 -o libbensolve.$dlext *.c -lglpk -lm
install -Dvm 755 libbensolve.$dlext -t ${libdir}
install -Dvm 644 *.h -t ${includedir}
install_license doc/manual.pdf
```
Isn't there a better license file than a PDF version of the manual?

File: L/libbensolve/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
+
```
Comment (giordano) [inline L/libbensolve/build_tarballs.jl:42]:
```suggestion
platforms = supported_platforms()
```

File: L/libbensolve/build_tarballs.jl (hunk 1 above)
Comment (kofgokhan) [reply L/libbensolve/build_tarballs.jl:19]: License is at the end of the manual but there is no other that ships with this library.

Comment (giordano) [reply L/libbensolve/build_tarballs.jl:19]: Use
```
install_license /usr/share/licenses/GPL-3.0+
```

File: L/libbensolve/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+cd $WORKSPACE/srcdir/bensolve*/
+cc -shared -fPIC -std=c99 -O3 -o libbensolve.$dlext *.c -lglpk -lm
+install -Dvm 755 libbensolve.$dlext -t ${libdir}
+install -Dvm 644 *.h -t ${includedir}
+install_license /usr/share/licenses/GPL-3.0+
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```
Comment (giordano) [inline L/libbensolve/build_tarballs.jl:24]: Congratulations, you're the first victim of #9393
```suggestion
platforms = supported_platforms(; exclude=p->Sys.isfreebsd(p) && arch(p) == "aarch64")
```

**Comment by giordano** [comment]
Thanks for your contribution!

**Comment by kofgokhan** [comment]
Thanks for the help!

**Comment by kofgokhan** [comment]
Sorry for this but this is supposed to be Bensolve-2.1.0 not libbensolve. My mistake. Is it OK if I rename this?

**Comment by giordano** [comment]
Ah, that's a great point, but I think the package is already registered, so changing the name at this point is just a hassle

**Pull request merged**